api/serializers.py

```diff
@@ -0,0 +1,34 @@
+from djoser.serializers import UserSerializer
+
+from users.models import User
+
+
+class CustomUserSerializer(UserSerializer):
```

Comment:
чтобы не добавлять ненужный префикс Custom можно сделать так
```suggestion
from djoser import serializers

from users.models import User


class UserSerializer(serializers.UserSerializer):
```